README.md

```diff
@@ -156,7 +155,6 @@ The [LogDNA API Key](https://app.logdna.com/manage/profile) associated with your
 * Type: [string][]
 * Default: `''`
 * Values: `YourCustomApp`
-* Max Length: `32`
```

Comment:
If `YourCustomApp` is a variable, let's use `<your custom app>`

Reply:
Done across all, @tanberry!